recipes/oscrypto/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "oscrypto" %}
+{% set version = "1.2.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 2f53f50813ecc5114fd126b89766fabdb9448e0bb080814451cc4bb92acafca4
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - asn1crypto >=1.0.0
+
+test:
+  requires:
+    - pip
+  imports:
+    - oscrypto
+  commands:
+    - pip check
```

Comment:
What is this for? I haven't seen in a conda recipe before?

Reply:
This checks for consistency of the Python dependencies. It might be actually useful to have this for all Python recipes in future, haven't yet had the time to start a discussion on it.